frontend/src/scenes/insights/insightVizDataLogic.ts

```diff
@@ -431,6 +431,7 @@ const handleQuerySourceUpdateSideEffects = (
      * Date range change side effects.
      */
     if (
+        !isPathsQuery(currentState) && // TODO: Apply side logic more elegantly
```

Comment:
What does this do?

Reply:
It added an interval even if there was none. And it's not supported by paths.